tests/checkpoint/test_checkpoint.py

```diff
@@ -1450,9 +1460,300 @@ def test_newstyle_checkpoint_instantiates_and_produces_a_validation_result_with_
+            data_context=context,
+            config_version=1,
+            run_name_template="%Y-%M-foo-bar-template",
+            expectation_suite_name="my_expectation_suite",
+            batch_request=batch_request,
+            validator=validator,
+            action_list=[
+                {
+                    "name": "store_validation_result",
+                    "action": {
+                        "class_name": "StoreValidationResultAction",
+                    },
+                },
+                {
+                    "name": "store_evaluation_params",
+                    "action": {
+                        "class_name": "StoreEvaluationParametersAction",
+                    },
+                },
+                {
+                    "name": "update_data_docs",
+                    "action": {
+                        "class_name": "UpdateDataDocsAction",
+                    },
+                },
+            ],
+        )
+
+
+@pytest.mark.slow  # 1.15s
+def test_newstyle_checkpoint_raises_error_if_batch_request_in_validations_and_validator_are_specified_in_constructor(
+    titanic_pandas_data_context_with_v013_datasource_with_checkpoints_v1_with_empty_store_stats_enabled,
+    batch_request_as_dict,
+):
+    context: DataContext = titanic_pandas_data_context_with_v013_datasource_with_checkpoints_v1_with_empty_store_stats_enabled
+    batch_request: BatchRequest = BatchRequest(**batch_request_as_dict)
+    context.add_expectation_suite("my_expectation_suite")
+    validator: Validator = context.get_validator(
+        batch_request=batch_request,
+        expectation_suite_name="my_expectation_suite",
+    )
+    with pytest.raises(
+        gx_exceptions.CheckpointError,
+        match=r'Checkpoint "my_checkpoint" cannot be called with a validator and contain a batch_request and/or a batch_request in validations.',
+    ):
+        _ = Checkpoint(
+            name="my_checkpoint",
+            data_context=context,
+            config_version=1,
+            run_name_template="%Y-%M-foo-bar-template",
+            expectation_suite_name="my_expectation_suite",
+            validator=validator,
+            action_list=[
+                {
+                    "name": "store_validation_result",
+                    "action": {
+                        "class_name": "StoreValidationResultAction",
+                    },
+                },
+                {
+                    "name": "store_evaluation_params",
+                    "action": {
+                        "class_name": "StoreEvaluationParametersAction",
+                    },
+                },
+                {
+                    "name": "update_data_docs",
+                    "action": {
+                        "class_name": "UpdateDataDocsAction",
+                    },
+                },
+            ],
+            validations=[{"batch_request": batch_request}],
+        )
+
+
+@pytest.mark.slow  # 1.15s
```

Comment:
And here

Reply:
Please address throughout

Reply:
@cdkini I addressed the `FileDataContext` type throughout; however, I am not sure if adding the "integration test" qualification should be in scope.  Thanks.

Reply:
@cdkini I took care of it -- thanks.